### ISSUE TITLE
Injected path cannot be marked as external: Vite 4.0.4

### ISSUE THREAD

**Issue by saravanabalagi** [voracious/vite-plugin-node-polyfills#11]
I added the lib using yarn and updated `vite.config.js` as instructed, but I get this error when I run the server:

```
✘ [ERROR] The injected path "myproj/node_modules/node-stdlib-browser/helpers/esbuild/shim.js" cannot be marked as external

Build failed with 1 error:
error: The injected path "myproj/node_modules/node-stdlib-browser/helpers/esbuild/shim.js" cannot be marked as external
```

I'm using
Vite: 4.0.4
Node: 18.12.1
Yarn: 1.22.19

**Comment by saravanabalagi** [comment]
Same as [this](https://github.com/sodatea/vite-plugin-node-stdlib-browser/issues/4#issuecomment-1365616424) issue from a similar lib that uses node-stdlib-browser

**Comment by voracious** [comment]
@saravanabalagi this is fixed in [`v0.7.0`](https://github.com/voracious/vite-plugin-node-polyfills/releases/tag/v0.7.0).
